test/data/keymaps/invalid-escape-sequence.xkb

```diff
@@ -0,0 +1,9 @@
+xkb_keymap {
+    // The following include statement has an octal escape sequence that
+    // must be ignored. Else it would insert a NULL character and thus
```

Comment:
NULL -> NUL

Reply:
Actually, the character name is NULL. `NUL` is the [ASCII abreviation](https://en.wikipedia.org/wiki/C0_and_C1_control_codes#ASCII). I prefer the name, but if you prefer the abbreviation then I suggest we use `\NUL` instead.